Makefile

```diff
@@ -24,6 +24,35 @@ ethereum-test: ethereum-clean
 ethereum-deploy: ethereum-clean
 	@./contracts/solidity/deploy.sh
 
+
+Command := $(firstword $(MAKECMDGOALS))
+PARAM := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+ethereum-set-escrow:
+ifneq ($(PARAM),)
+	@./contracts/solidity/set_escrow.sh $(PARAM)
+else
+	@echo "Error: New Escrow address nedded"
+	@echo "Example of usage:"
+	@echo "make ethereum-set-escrow 0x01234..."
+endif
+%::
+	@true
+
+
+Command := $(firstword $(MAKECMDGOALS))
+PARAM := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+ethereum-set-withdraw-selector:
+ifneq ($(PARAM),)
+	@./contracts/solidity/set_withdraw_selector.sh $(PARAM)
+else
+	@echo "Error: New withdraw selector nedded"
+	@echo "Example of usage:"
+	@echo "make ethereum-set-withdraw-selector 0x01234..."
+endif
+%::
+	@true
```

Comment:
Uri, should we run these commands immediately after deploying the Starknet contract, right? Could we consider adding them directly to the deploy.sh?

Reply:
done with set-escrow , but i'm not sure which value to pass to set-withdraw-selector, should i execute it with our current value hardcoded?

Reply:
I would set that value or retrieve it from an environment variable. I think that it's something that not will change often, in fact, it's most likely not to change.

Reply:
I think is better to not have them as an env variable because it's data we don't know until the whole deployment starts

Reply:
Yes, the withdraw_selector can be pre-calculated

Reply:
that's true only after you do the compilation

Reply:
Are you sure?
I think you can use https://www.stark-utils.xyz/converter to calculate a function's selector beforehand, as stated in the Readme

Reply:
This thread has turned obsolete since commit 32abb05d8c8c9aa421612de4034f778177fae05f